src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/OnTypeFormattingStructureValidationPass.cs

```diff
@@ -1,172 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Diagnostics;
-using System.Linq;
-using Microsoft.AspNetCore.Razor.Language;
-using Microsoft.AspNetCore.Razor.Language.Syntax;
-using Microsoft.AspNetCore.Razor.LanguageServer.Common;
-using Microsoft.Extensions.Logging;
-using OmniSharp.Extensions.LanguageServer.Protocol.Models;
-using OmniSharp.Extensions.LanguageServer.Protocol.Server;
-
-namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting
-{
-    internal class OnTypeFormattingStructureValidationPass : FormattingPassBase
```

Comment:
So this was just restricting where the ontype formatting would occur. The pre-existing structure validation pass that validates that no content has been "destroyed" still exists right?

Reply:
>  pre-existing structure validation

The other validation we currently have are,
- Content validation (makes sure only whitespace is nuked)
- Diagnostic validation (makes sure we don't introduce new diagnostics. This works surprisingly well in most cases)